### PR TITLE
archiving notes

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

- Adding notes from 19 June 2023 & 26 June 2023

### CODE REVIEW

File: governance/community-calls/20230626-coworking.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Kirstie
+* Anne
+* Emma
+* Vicky
+* Bastian
+* Gabin
+* Eirini
+* David
+* Alex
+* Batool,
```
Comment (AlexandraAAJ) [inline governance/community-calls/20230626-coworking.md:14]: Remove the "," after Batool's name

File: governance/community-calls/20230626-coworking.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Topics/Rooms
+
+* Fireside Chat
+    * Anne (if Alejandro & Alexandra are here!)
+* Open Science EU
+    * (If time) - Emma, Anne, Gabin
+* Governance (main room)
+  * Kirstie
+  * Alden (Margaret Hamilton)
```
Comment (AlexandraAAJ) [inline governance/community-calls/20230626-coworking.md:27]: Add: Malvika, Anne and Alex

**Review by AlexandraAAJ — COMMENTED** [review]
Proposing some tiny changes to the notes

File: governance/community-calls/20230626-coworking.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
```
Comment (aleesteele) [inline governance/community-calls/20230626-coworking.md:27]:
```suggestion
  * Alden (Margaret Hamilton)
  * Alexandra
  * Anne
  * Malvika
```

File: governance/community-calls/20230626-coworking.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  * Alexandra
+  * Anne
+  * Malvika
+*
```
Comment (aleesteele) [inline governance/community-calls/20230626-coworking.md:31]:
```suggestion
```

**Comment by aleesteele** [comment]
Thank you @AlexandraAAJ! 🙏 Corrected these errors.

File: governance/community-calls/20230619-coworking.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Notes 19 June
```
Comment (aleesteele) [inline governance/community-calls/20230619-coworking.md:1]:
```suggestion
# Notes 19 July
```

File: governance/community-calls/20230626-coworking.md (hunk 1 above)
Comment (aleesteele) [inline governance/community-calls/20230626-coworking.md]:
```suggestion
* Batool
```

**Review by EstherPlomp — APPROVED** [review]
Looks good! Do we maybe want to remove empty bullet points?

**Comment by aleesteele** [comment]
Good call @EstherPlomp! I removed extra bullet points that disrupt an ongoing list, but left the empty lists as is.

**Comment by aleesteele** [comment]
Thanks @EstherPlomp and @AlexandraAAJ - merging now!

**Pull request merged**